gatling/user-files/DockerFile/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM denvazh/gatling
+
+RUN apk add nodejs-current nodejs-npm && \
```

Comment:
Not mandatory but you should install Node.js from tarball instead of Alpine package repository.